src/pipit/settings/prod.py

```diff
@@ -33,6 +33,19 @@
     },
 }
 
+MIDDLEWARE += [
+    "opencensus.ext.django.middleware.OpencensusMiddleware",
+]
+APPLICATION_INSIGHTS_CONNECTION_STRING = get_env("APPLICATION_INSIGHTS_CONNECTION_STRING")
+OPENCENSUS = {
+    "TRACE": {
+        "SAMPLER": "opencensus.trace.samplers.ProbabilitySampler(rate=1)",
```

Comment:
We hebben al Sentry om te tracen.

- Voegt dit functioneel iets toe? 
- Hoe duur is het als we alles gaan trace met rate=1? Bij Sentry zaten we al snel tegen de limieten omdat hij ook de database queries opstuurt.
- Kunnen we een Sampler hebben die werkt a.d.h.v een cookie en/of query parameter net zoals we doen voor Sentry?

Reply:
Misschien is het dan veel makkelijker om https://docs.sentry.io/platforms/javascript/guides/nextjs/ toe te voegen voor monitoring van de frontend. Ik zag namelijk dat google analytics ook al gebruikt werd maar geen idee of iemand hiernaar kijkt. Ik ga er even achteraan want een oplossing Sentry voor logging, Analytics voor statistieken is dan logischer